src/System.Collections.NonGeneric/tests/System.Collections.NonGeneric.Tests.csproj

```diff
@@ -8,6 +8,8 @@
     <RootNamespace>System.Collections.NonGeneric.Tests</RootNamespace>
     <AssemblyName>System.Collections.NonGeneric.Tests</AssemblyName>
     <ProjectGuid>{EE95AE39-845A-42D3-86D0-8065DBE56612}</ProjectGuid>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
+    <TargetFrameworkProfile />
```

Comment:
The problem is that you can't even build the test project at all, let alone run the tests, if you don't have VS 2015 installed. Like @ianhays was saying, though, we could probably only include the performance source files if you have Performance=true in your build settings. You would still get the warning about referencing the v5.0 portable project (because it's pulled in by the project.json file), but it would still be able to build the rest of the source files.